content_scripts/CameraVis.js

```diff
@@ -87,7 +88,7 @@ void main() {
       ((b & (1 << 7)) << (23 - 7))) *
       (5000.0 / float(1 << 24));
   `}
-  float z = depth - 1.0;
+  float z = (depth - 1.0) * patchLoading;
```

Comment:
I'm curious how this effect will actually look like. I thought it would be some progress along depth. But it is also affecting x, y below.

Reply:
Posted a video in Teams of what it looks like. It's sort of like a camera flash shooting out from the CameraVis